### PR TITLE
hyperthreading Fix copy/paste issue

### PR DESCRIPTION
**Pull request by nraynaud**

### CODE REVIEW

**Comment by stormi** [comment]
You should rebase your branch (or create a new one) on master first, here the PR shows 4 commits.

File: SOURCES/etc/xapi.d/plugins/hyperthreading.py [hunk 1]
```diff
@@ -12,20 +12,11 @@
...
-        result = run_command(['xl', 'info', 'threads_per_core'])
-        _LOGGER.info(result)
-        lines = result['stdout'].splitlines()
-
-        return json.dumps(int(lines[0]) > 1)
-    except OSError as e:
-        if e.errno == errno.ENOENT:
-            return json.dumps({})
-        else:
-            raise e
```
Comment (julien-f) [inline SOURCES/etc/xapi.d/plugins/hyperthreading.py]: Is there a reason for the `try`/`expect` to be removed?

Comment (nraynaud) [reply SOURCES/etc/xapi.d/plugins/hyperthreading.py]: yes, it was a leftover copy/paste from another plugin. Here it makes no sense. If the plugin is installed and the `xl` is not present we have a big problem and we need to investigate. So I let the error through.

**Comment by julien-f** [comment]
I only left a single comment, I don't know why GitHub shows two :confused:

**Comment by stormi** [comment]
Then I'll update the commit message myself to avoid another round-trip.

**Comment by stormi** [comment]
Commit message updated and commit applied to master.

**Pull request closed without merging**